root/static/scripts/work.js

```diff
@@ -70,6 +70,11 @@ const store = createStore(function (state: WorkForm = form, action) {
         action.languageId,
         state,
       );
+      if (_.intersection(state.field.languages.field.map(lang => lang.value), ["486", "284"]).length > 0) {
+        state.field.languages.errors = 1;
+      } else {
+        state.field.languages.errors = 0;
+      }
```

Comment:
It is a bit weird to set a number to `errors` which it is supposed to hold a string array. I suggest to move the condition (on line 73) into a function to be passed to `FormRowSelectList` as a property (see the latest suggested type in previous comment).